### ISSUE TITLE
Option to keep sidebar in Sites 

### ISSUE THREAD

**Issue by ingjerd123** [Simplero/Roadmap#225]
Hi, 

This is about the sidebar in sites

I want to hide a page from the main navigation. But when I do that, the sidebar for that page and all its sub pages disappears, so all the pages end up looking too wide and funny. Could it be possible to chose which pages to have sidebars and not, even when the box "hide from main navigation" is ticked? If not, maybe a possibility to make some pages narrower like if they had a sidebar? I don't want to rebuild the pages somewhere else, because then I would lose the Facebook comments

This is the main page I want to hide to give you a better understanding: https://norskgrammatikk.simplero.com/pages/nyhetsbrev

**Comment by AndreaSchroeder77** [comment]
Yes this would be helpful for me too. Since moving to the membership site I am getting a lot of comments about the site being so wide on pages without sidebars, making the text harder to read.
